dsc/workflows/base/simple_csv.py

```diff
@@ -56,7 +56,7 @@ def reconcile_bitstreams_and_metadata(
             s3_client.files_iter(
                 bucket=self.s3_bucket,
                 prefix=self.batch_path,
-                exclude_prefixes=["archived", metadata_file],
+                exclude_prefixes=["archived", metadata_file, "metadata.json"],
```

Comment:
As discussed, I think this is a good option for now.  Granted this will prevent a record in DSpace from having a `metadata.json` bitstream, it's a good solution for now.